lib/matplotlib/colors.py

```diff
@@ -121,6 +121,12 @@ def is_color_like(c):
         return True
 
 
+def has_alpha_channel(c):
```

Comment:
do we really want this as public?  

Otherwise this looks fine to me...

Reply:
I'd say it's ok as the API is quite obvious (no chance of getting something wrong that we regret later) and it's simple enough that it's not a maintenance burden. OTOH, it should be quite rare that users need this. So let's be defensive and not make it public for now.